ai_council/execution/worker.py

```
@@ -0,0 +1,174 @@
+                        "token_usage": sa.token_usage if sa else 0,
+                        "execution_time": sa.execution_time if sa else 0.0,
+                        "model_used": sa.model_used if sa else ""
+                    } if sa else {}
+                }
+                
+                # Publish the final result
+                await self.redis_client.rpush(response_key, json.dumps(result_payload))
+                logger.info(f"Worker completed subtask {subtask.id} successfully")
+                current_span.set_attribute("success", True)
+
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)